lib/lens/core.rb

```diff
@@ -5,3 +5,4 @@
 require "lens/gc"
 require "lens/trace"
 require "lens/worker"
+require "lens/allocations_data"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.